api/resources.ts

```diff
@@ -1,16 +1,16 @@
 import { AirtableClient } from './airtable'
 import { 
   Service,
-  ServicesLists,
+  ServicesList,
  } from '../models'
 import { keys } from 'ts-transformer-keys'
 
-const resourcesClient = new AirtableClient(process.env.RESOURCES_API_KEY, process.env.RESOURCES_BASE_ID)
+const resourcesClient = new AirtableClient(process.env.NEXT_PUBLIC_RESOURCES_API_KEY, process.env.NEXT_PUBLIC_RESOURCES_BASE_ID)
```

Comment:
According to the [Next.js docs](https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser), we need to prefix our environment variables with `NEXT_PUBLIC_` to access them anywhere in our code.

Without it, they're only available in the Node.js environment, which I think means they can only be used in certain Next.js methods described [here](https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables).